### PR TITLE
ecs run launcher add hooks to change run tags

### PR DESCRIPTION
**Pull request by prha**
### Summary & Motivation
Enables subclasses to change the format of the ECS info stored in tags.

### How I Tested These Changes
BK

### CODE REVIEW

**Review by gibsondan — CHANGES_REQUESTED** [review]
we ended up doing this differently i think?

**Comment by prha** [comment]
This is separate.... in case we want to change the stored run tags to remove the serverless account id from the arn/cluster.

File: python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py [hunk 1]
```diff
@@ -164,20 +164,21 @@ def _set_ecs_tags(self, run_id, task_arn):
...
 
-    def _set_run_tags(self, run_id, task_arn):
+    def set_run_tags(self, run_id, task_arn):
+        # Extracted method to allow for subclasses to customize the ECS tag storage implementation
         cluster = self._task_metadata().cluster
         tags = {"ecs/task_arn": task_arn, "ecs/cluster": cluster}
         self._instance.add_run_tags(run_id, tags)
 
-    def _get_run_tags(self, run_id):
+    def get_ecs_info(self, run_id) -> EcsInfo:
```
Comment (gibsondan) [inline python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py:173]: bit confused why we took out the underscore - them being overridden by a subclass doesn't necc make them public I think?

Comment (gibsondan) [reply python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py:173]: def _get_ecs_info_from_run_tags?

**Review by gibsondan — APPROVED** [review]
better late than never

**Pull request closed without merging**